composer.json

```diff
@@ -18,10 +18,10 @@
     "require-dev": {
         "phpunit/phpunit": "^6 || ^7 || ^8 || ^9",
         "magento/magento-coding-standard": "*",
-        "dealerdirect/phpcodesniffer-composer-installer": "dev-master",
+        "dealerdirect/phpcodesniffer-composer-installer": "v1.0.0",
         "phpstan/phpstan": "^0.12.2 || ^1.2",
         "phpstan/extension-installer": "1.2.x-dev",
-        "bitexpert/phpstan-magento": "dev-master"
+        "bitexpert/phpstan-magento": "v0.28.0"
```

Comment:
@aashwin-rvvup do you want to lock this down to `0.28.0` ? wondering if `~0.28` would do the trick. It would lock it down to any pre version `1.0` release. It would need changing to `~0.28 || ^1.0` when version 1 of that library is released